.vscode/tasks.json

```diff
@@ -0,0 +1,20 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "cppbuild",
+			"label": "Build for Debugging",
```

Comment:
Would it also make sense to have a task to build for "production"?

Reply:
Hmmm, sure, I don't see why not, I will make one!

Reply:
Created a new task for prod build